apps/els_lsp/src/els_fungraph.erl

```diff
@@ -0,0 +1,37 @@
+-module(els_fungraph).
+
+-type gnode() :: #{id := _ID, _ => _}.
+-type edges_fun() :: fun((node()) -> list(node())).
```

Comment:
this might be a typo? `node() -> gnode()`

Reply:
Oops, you're right. Thanks for a quick review!